doc/man3/SSL_CTX_use_serverinfo.pod

```diff
@@ -2,33 +2,59 @@
 
 =head1 NAME
 
-SSL_CTX_use_serverinfo, SSL_CTX_use_serverinfo_file - use serverinfo extension
+SSL_CTX_use_serverinfo_ex,
+SSL_CTX_use_serverinfo,
+SSL_CTX_use_serverinfo_file
+- use serverinfo extension
 
 =head1 SYNOPSIS
 
  #include <openssl/ssl.h>
 
+ int SSL_CTX_use_serverinfo_ex(SSL_CTX *ctx, unsigned int version,
+                               const unsigned char *serverinfo,
+                               size_t serverinfo_length);
+
  int SSL_CTX_use_serverinfo(SSL_CTX *ctx, const unsigned char *serverinfo,
                             size_t serverinfo_length);
 
  int SSL_CTX_use_serverinfo_file(SSL_CTX *ctx, const char *file);
 
 =head1 DESCRIPTION
 
-These functions load "serverinfo" TLS ServerHello Extensions into the SSL_CTX.
-A "serverinfo" extension is returned in response to an empty ClientHello
+These functions load "serverinfo" TLS extensions into the SSL_CTX. A
+"serverinfo" extension is returned in response to an empty ClientHello
 Extension.
```

Comment:
No - this applies to all protocol versions. "serverinfo" is an existing capability which we're just upgrading to be able to cope with TLS1.3 too.